content/games/factions/rules.md

```diff
@@ -246,6 +246,23 @@ Well-Armed (1)
 |  Fortify   |  7+   |  18”  |    Another Group    | Until the start of your next turn. | Target may force enemies to reroll any number of to-hit dice once against them. |
 |  Counter   |  8+   |  18”  |        Group        |                 -                  | End any Spell effect on the target as if its duration ended.                    |
 
+#### Taunts
+
+| Taunt                          | Check | Requirements                                     | Effect                                                                                                                                 |
+| :----------------------------- | :---: | :----------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------- |
+| Come and Die!                  |  6+   | -                                                | Cannot Move or Retreat (-1 FS if they would otherwise do so) and all enemy Groups are Reckless against them.                           |
```

Comment:
```suggestion
| Come and Die!                  |  6+   | -                                                | Cannot Move or Retreat (-1 FS instead if they would be forced to do so), and all enemy Groups are Reckless against them.                           |
```

Reply:
Should that comma be there or should it read:
> Cannot Move or Retreat (-1 FS instead if they would be forced to do so) and all enemy Groups are Reckless against them.
```suggestion
| Come and Die!                  |  6+   | -                                                | Cannot Move or Retreat (-1 FS instead if they would be forced to do so) and all enemy Groups are Reckless against them.                           |
```

Reply:
either is fine actually. now im looking at it, the ethos of this doc probably leans toward your suggestion.